unicodetools/org/unicode/tools/emoji/GenerateEmoji.java

```diff
@@ -3490,9 +3490,10 @@ static void showCandidateStyle(CandidateStyle candidateStyle, String outFileName
 		+ "</ul>\n"
 		: "";
 	String top4 = candidateStyle != CandidateStyle.released
-		? "Where the last Sample cell is marked with the symbol "
-		+ NOT_NEW + " it indicates that a previously existing character is included, "
-		+ "and only because it has new gender and/or modifier variants.</p>\n"
+		? " Where the last Sample cell is marked with the symbol "
+		+ NOT_NEW + ", there is a row for an pre-existing character that has new gender and/or modifier variants. "
+		+ "For example, ➕ 5 Ⓒ ‧ 🏿 means that additional 5 skintone variants on the original character are added,  "
```

Comment:
Helpful addition here for those who need to make sense of the emoji counts keys!